.circleci/config.yml

```diff
@@ -443,6 +443,11 @@ jobs:
     - run:
         name: Install Kubectl
         command: make dev/install/kubectl
+    - run:
+        name: Install Conntrack
```

Comment:
What's that? Why we need it? Should we integrate this with our makefiles?

Reply:
I don't know, for some reason on Ubuntu with `--vm-driver=None` minikube wants `conntrack` in its path (only for kubernetes 1.18.x). We don't have makefile target that setup minikube (like we have for kind) so I'm not sure we need `install/conntrack` as target

Reply:
Ok, it seems we don't need it in makefiles. You can drop a comment here that it is required by minikube with `--vm-driver=None`